<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Игра: Угадай число</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelInstruction">
      <property name="text">
       <string>Угадайте число от 1 до 100:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditInput"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonGuess">
      <property name="text">
       <string>Проверить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonReset">
      <property name="text">
       <string>Сбросить игру</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelFeedback">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="listViewHistory"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
